### DISCUSSION TITLE
Can not select multiple layers in Navigator

### DISCUSSION THREAD

**Discussion by iroshandezilva** [webstudio-is/webstudio-community#42] [Wishlist]
Recently i want to reorder few layers, I found that we aren't support for `Shit+click` to select multiple layers.

**Comment by kof** [comment]
That's right, Webflow doesn't support it either, but we should do better and add it.

**Reply by silasrogan** [reply]
Is this something that's still planned (perhaps in combination with a right click context menu)? I'll often create elements to serve as the scaffold for variants and being able to multi-select is crucial for this.

**Reply by kwameandco** [reply]
Love this attitude. It would be great for speeding up workflows!
